sdk/tables/azure-data-tables/src/test/java/com/azure/data/tables/TableServiceAsyncClientTest.java

```diff
@@ -87,6 +86,51 @@ public void serviceCreateTable() {
             .verify();
     }
 
+
+    @Test
+    public void serviceAsyncClientCreateTableFailsWithIllegalArgumentExceptionIfTableNameInURI() {
```

Comment:
nit; The convention we follow in the Java SDK is that when writing using camelCase we don't make a special distinction for acronyms. Thus, `URI` ends up being `uri` or `Uri` depending on where it is in the string.
```suggestion
    public void serviceAsyncClientCreateTableFailsWithIllegalArgumentExceptionIfTableNameInUri() {
```

Reply:
Ok, good to know